en.json

```diff
@@ -1,12 +1,12 @@
 {
   "Learn More" : "Learn More",
   "CLOSED CAPTION PREVIEW": "CLOSED CAPTION PREVIEW",
-  "Sample Text": "Sample Text",
+  "Sample Text": "Hi, I'm a caption!",
```

Comment:
I figured to just leave it as translated "Sample Text" for now until confirmed that we want this text.